build.gradle

```diff
@@ -83,6 +84,9 @@ test {
     }
 }
 
+
```

Comment:
Please remove this additional empty lines.

Reply:
removed!